### PR TITLE
MNT Remove TODO comments

### PR DESCRIPTION
**Pull request by sabina-talipova**
## Parent issue
- https://github.com/silverstripe/.github/issues/116

### CODE REVIEW

File: client/src/components/UploadField/UploadField.js
Comment (sabina-talipova) [inline client/src/components/UploadField/UploadField.js:1]: - https://github.com/silverstripe/silverstripe-asset-admin/issues/1406
- https://github.com/silverstripe/silverstripe-asset-admin/issues/1407
- https://github.com/silverstripe/silverstripe-asset-admin/issues/1408
- https://github.com/silverstripe/silverstripe-asset-admin/issues/1409
- https://github.com/silverstripe/silverstripe-asset-admin/issues/1410
- https://github.com/silverstripe/silverstripe-asset-admin/issues/1411
- https://github.com/silverstripe/silverstripe-asset-admin/issues/1412

File: client/src/containers/AssetAdmin/AssetAdmin.js [hunk 1]
```diff
@@ -283,8 +283,6 @@ class AssetAdmin extends Component {
           this.handleOpenFile(response.record.id);
         }
 
-        // TODO Update GraphQL store with new model,
-        // see https://github.com/silverstripe/silverstripe-graphql/issues/14
```
Comment (sabina-talipova) [inline client/src/containers/AssetAdmin/AssetAdmin.js:287]: - https://github.com/silverstripe/silverstripe-asset-admin/issues/1406

File: client/src/containers/AssetAdmin/AssetAdmin.js [hunk 2]
```diff
@@ -477,9 +475,6 @@ class AssetAdmin extends Component {
    */
   handleUnpublish(fileIds) {
     return this.doUnpublish(fileIds).then((response) => {
-      // TODO Update GraphQL store with new model or update apollo and use new API
-      // see https://github.com/silverstripe/silverstripe-graphql/issues/14
-      // see https://dev-blog.apollodata.com/apollo-clients-new-imperative-store-api-6cb69318a1e3
```
Comment (sabina-talipova) [inline client/src/containers/AssetAdmin/AssetAdmin.js:482]: - https://github.com/silverstripe/silverstripe-asset-admin/issues/1406

File: client/src/containers/AssetAdmin/AssetAdmin.js [hunk 3]
```diff
@@ -558,7 +552,6 @@ class AssetAdmin extends Component {
   }
 
   handleMoveFilesSuccess(folderId, fileIds) {
-    // TODO Refactor "queued files" into separate visual area and remove coupling here
```
Comment (sabina-talipova) [inline client/src/containers/AssetAdmin/AssetAdmin.js:561]: - https://github.com/silverstripe/silverstripe-asset-admin/issues/1407

File: client/src/containers/AssetAdmin/AssetAdmin.js [hunk 4]
```diff
@@ -799,7 +790,6 @@ function mapStateToProps(state, ownProps) {
   const { formSchema } = state.assetAdmin.modal;
   return {
     securityId: state.config.SecurityID,
-    // TODO Refactor "queued files" into separate visual area and remove coupling here
```
Comment (sabina-talipova) [inline client/src/containers/AssetAdmin/AssetAdmin.js:802]: - https://github.com/silverstripe/silverstripe-asset-admin/issues/1407

File: client/src/containers/AssetAdmin/AssetAdmin.js [hunk 5]
```diff
@@ -812,7 +802,6 @@ function mapDispatchToProps(dispatch) {
       gallery: bindActionCreators(galleryActions, dispatch),
       toasts: bindActionCreators(toastsActions, dispatch),
       displaySearch: bindActionCreators(displaySearchActions, dispatch),
-      // TODO Refactor "queued files" into separate visual area and remove coupling here
```
Comment (sabina-talipova) [inline client/src/containers/AssetAdmin/AssetAdmin.js:815]: - https://github.com/silverstripe/silverstripe-asset-admin/issues/1407

File: client/src/containers/Gallery/Gallery.js
Comment (sabina-talipova) [inline client/src/containers/Gallery/Gallery.js:1]: - https://github.com/silverstripe/silverstripe-asset-admin/issues/1408

File: client/src/containers/HistoryList/HistoryItem.js [hunk 6]
```diff
@@ -25,7 +25,6 @@ class HistoryItem extends Component {
     }
 
     return (
-      // @todo wrap the contents in an `<a>` tag
```
Comment (sabina-talipova) [inline client/src/containers/HistoryList/HistoryItem.js:28]: - https://github.com/silverstripe/silverstripe-asset-admin/issues/1411

File: client/src/containers/HistoryList/HistoryList.js [hunk 7]
```diff
@@ -47,7 +45,7 @@ class HistoryList extends Component {
   }
 
   componentDidUpdate(prevProps) {
-    // TODO race conditions happening, this should have history state shifted to redux
```
Comment (sabina-talipova) [inline client/src/containers/HistoryList/HistoryList.js:50]: - https://github.com/silverstripe/silverstripe-asset-admin/issues/1412

File: client/src/containers/AssetAdmin/AssetAdmin.js
Comment (sabina-talipova) [inline client/src/containers/AssetAdmin/AssetAdmin.js:1]: - https://github.com/silverstripe/silverstripe-asset-admin/issues/1410

File: client/src/entwine/TinyMCE_ssmedia.js [hunk 8]
```diff
@@ -271,10 +271,6 @@ jQuery.entwine('ss', ($) => {
     _handleInsert(data, file) {
       let result = false;
       this.setData(Object.assign({}, data, file));
-
-      // Sometimes AssetAdmin.js handleSubmitEditor() can't find the file
-      // @todo Ensure that we always return a file for any valid ID
-
```
Comment (sabina-talipova) [inline client/src/entwine/TinyMCE_ssmedia.js:277]: - https://github.com/silverstripe/silverstripe-asset-admin/issues/1410

File: client/src/state/files/moveFilesMutation.js [hunk 9]
```diff
@@ -23,10 +23,6 @@ const config = {
             fileIds,
           },
           update: () => {
-            // todo:
-            // Refactor this once Apollo GraphQL adds support
-            // for invalidating specific queries in the cache.
-            // Context: https://github.com/silverstripe/silverstripe-asset-admin/issues/809
```
Comment (sabina-talipova) [inline client/src/state/files/moveFilesMutation.js:29]: - https://github.com/silverstripe/silverstripe-asset-admin/issues/1409

File: client/src/containers/HistoryList/HistoryList.js [hunk 10]
```diff
@@ -47,7 +45,7 @@ class HistoryList extends Component {
...
+    // race conditions happening, this should have history state shifted to redux
```
Comment (ChloeSartorelli) [inline client/src/containers/HistoryList/HistoryList.js]:
```suggestion
    // Avoids race conditions from happening
```

File: code/Extensions/RemoteFileModalExtension.php [hunk 11]
```diff
@@ -104,7 +104,7 @@ public function remoteEditFormSchema(HTTPRequest $request)
         } catch (NetworkException | RequestException | InvalidRemoteUrlException $exception) {
             $errors = ValidationResult::create()
                 ->addError($exception->getMessage());
-            // @todo - Don't create dummy form (pass $form = null)
+            // Don't create dummy form (pass $form = null)
```
Comment (ChloeSartorelli) [inline code/Extensions/RemoteFileModalExtension.php]:
```suggestion
```
That was part of the todo comment.

File: tests/behat/src/FixtureContext.php [hunk 12]
```diff
@@ -89,7 +89,7 @@ public function iShouldSeeTheForm($not, $id, $timeout = 3)
      */
     public function iShouldSeeTheFileStatusFlag()
     {
-        // TODO This should wait for any XHRs and React rendering to finish
+        // This should wait for any XHRs and React rendering to finish
```
Comment (ChloeSartorelli) [inline tests/behat/src/FixtureContext.php]:
```suggestion
```
That was part of the todo comment. It _should_ wait, but it doesn't.

File: tests/behat/src/FixtureContext.php [hunk 13]
```diff
@@ -106,7 +106,7 @@ public function iShouldSeeTheFileStatusFlag()
      */
     public function iShouldNotSeeTheFileStatusFlag()
     {
-        // TODO Flakey assertion, since the status flag might not be loaded via XHR yet
+        // Flakey assertion, since the status flag might not be loaded via XHR yet
```
Comment (ChloeSartorelli) [inline tests/behat/src/FixtureContext.php]:
```suggestion
```
That was part of the todo comment.

**Review by ChloeSartorelli — CHANGES_REQUESTED** [review]

File: client/src/containers/HistoryList/HistoryList.js (hunk 10 above)
Comment (sabina-talipova) [reply client/src/containers/HistoryList/HistoryList.js]: Updated

File: code/Extensions/RemoteFileModalExtension.php (hunk 11 above)
Comment (sabina-talipova) [reply code/Extensions/RemoteFileModalExtension.php]: Updated

File: tests/behat/src/FixtureContext.php (hunk 12 above)
Comment (sabina-talipova) [reply tests/behat/src/FixtureContext.php]: Updated

File: tests/behat/src/FixtureContext.php (hunk 13 above)
Comment (sabina-talipova) [reply tests/behat/src/FixtureContext.php]: Updated

**Review by ChloeSartorelli — APPROVED** [review]
LGTM

**Pull request merged**